src/central-server/admin-service/ui-system-test/src/intTest/java/org/niis/xroad/cs/test/ui/glue/CommonStepDefs.java

```diff
@@ -40,4 +59,10 @@ public void clickDialogSave() {
     public void statusDialogSave(ParameterMappers.SelenideValidation selenideValidation) {
         commonPageObj.dialog.btnSave().shouldBe(selenideValidation.getSelenideCondition());
     }
+
+    @ParameterType("SYSTEM_ADMINISTRATOR|REGISTRATION_OFFICER|SECURITY_OFFICER|MANAGEMENT_SERVICE")
```

Comment:
Currently the token type might only be the REGISTRATION_OFFICER. The TokenType enum doesn't have other values.